dotCMS/src/main/java/com/dotmarketing/servlets/StartupLogger.java

```diff
@@ -0,0 +1,65 @@
+package com.dotmarketing.servlets;
+
+import java.util.TimeZone;
+
+import com.dotcms.enterprise.LicenseUtil;
+import com.dotcms.enterprise.license.LicenseManager;
+import com.dotmarketing.business.CacheLocator;
+import com.dotmarketing.business.ChainableCacheAdministratorImpl;
+import com.dotmarketing.cms.factories.PublicCompanyFactory;
+import com.dotmarketing.db.DbConnectionFactory;
+import com.dotmarketing.db.HibernateUtil;
+import com.dotmarketing.exception.DotHibernateException;
+import com.dotmarketing.util.Config;
+import com.dotmarketing.util.Logger;
+import com.dotmarketing.util.UtilMethods;
+import com.liferay.portal.model.Company;
+
+public class StartupLogger {
+
+    public void log(){
+        
+        LicenseManager license = LicenseManager.getInstance();
+        
+        
+        
+        Company company = PublicCompanyFactory.getDefaultCompany();
+        TimeZone companyTimeZone = company.getTimeZone();
+        TimeZone.setDefault(companyTimeZone);
+        Logger.info(this, "InitServlet: Setting Default Timezone: " + companyTimeZone.getDisplayName());
+
+        String _dbType = DbConnectionFactory.getDBType();
+        String _dailect = "";
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Rename this local variable to match the regular expression '^[a-z][a-zA-Z0-9]*$'. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS00117)